tensorrt_llm/_torch/custom_ops/cpp_custom_ops.py

```
@@ -55,15 +55,11 @@ def _(
         else:
             return [torch.empty_like(input)]
 
-    #MNNVL Allreduce
-    @torch.library.register_fake("trtllm::mnnvl_twoshot_allreduce")
-    def _(input, buffer, buffer_flags, buffer_size, wait_for_results):
+    # MNNVL Allreduce
+    @torch.library.register_fake("trtllm::mnnvl_fusion_allreduce")
+    def _(input, residual, gamma, epsilon, buffer, buffer_flags,
```

Ruff (0.13.1)

Unused function argument: `gamma`

(ARG001)